_data-prepper/pipelines/configuration/processors/parse-json.md

```diff
@@ -24,65 +23,68 @@ This table is autogenerated. Do not edit it.
-| `parse_when` | No | String | Specifies under which conditions the processor should perform parsing. Default is no condition. Accepts an OpenSearch Data Prepper expression string following the [expression syntax]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/expression-syntax/). |
-| `overwrite_if_destination_exists` | No | Boolean | Overwrites the destination if set to `true`. Set to `false` to prevent changing a destination value that exists. Defaults to `true`. |
-| `delete_source` | No | Boolean | If set to `true` then this will delete the source field. Defaults to `false`. |
-| `tags_on_failure` | No | String | A list of strings specifying the tags to be set in the event that the processor fails or an unknown exception occurs during parsing.
+| `source` | No | String | The field in the event that will be parsed. Default is `message`. |
+| `destination` | No | String | The destination field for the parsed JSON. Defaults to the root of the event. Cannot be `""`, `/`, or any white-space-only string. |
+| `pointer` | No | String | A JSON pointer (as defined by [RFC 6901](https://datatracker.ietf.org/doc/html/rfc6901)) to a specific field in the source JSON. If omitted, the entire `source` is parsed. If the pointer is invalid, the full `source` is parsed instead. When writing to the root destination, existing keys will be preserved unless overwritten. |
+| `parse_when` | No | String | A condition expression that determines when to parse the field. Accepts a string following the [expression syntax]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/expression-syntax/). |
+| `overwrite_if_destination_exists` | No | Boolean | Whether to overwrite the destination field if it already exists. Default is `true`. |
+| `delete_source` | No | Boolean | Whether to delete the source field after parsing. Default is `false`. |
+| `tags_on_failure` | No | String | A list of tags to apply if parsing fails or an unexpected exception occurs. |
 
 ## Usage
 
-To get started, create the following `pipeline.yaml` file:
+To use the `parse_json` processor, add it to your `pipeline.yaml` configuration file:
 
 ```yaml
 parse-json-pipeline:
   source:
     ...
-  ....  
+  ...
   processor:
     - parse_json:
 ```
 
 ### Basic example
 
-To test the `parse_json` processor with the previous configuration, run the pipeline and paste the following line into your console, then enter `exit` on a new line:
+This example parses a JSON message field and flattens the data into the event.
 
-```
+For example, the following JSON message contains a key-value pair:
+
+```json
 {"outer_key": {"inner_key": "inner_value"}}
 ```
 {% include copy.html %}
 
-The `parse_json` processor parses the message into the following format:
+In example event, the original `message` field remains, and the parsed content is added at the root level. Use the `delete_source` option if you want to remove the original field:
```

Comment:
>  Use the `delete_source` option if you want to remove the original field:

The example below includes the original. So putting this sentence about `delete_source` is a little confusing in the context.

I recommend either of these changes:
* Drop the sentence about `delete_source`; or
* Create a new paragraph below that has this sentence and then show the result (which would not have `message`).